src/app/dynamic-forms/service/print/print.service.ts

```diff
@@ -123,11 +123,16 @@ export class PrintService {
   }
 
   async getDocDefFromForm(form) {
+    if(form!=null){
```

Comment:
It's cleaner to do a:

`if (form == null)`
`return;`

at the beginning instead of wrapping the entire block of code in the function in the if statement

Reply:
Resolved